libbeat/publisher/bulk.go

```diff
@@ -74,6 +74,10 @@ func (b *bulkWorker) onEvent(signal outputs.Signaler, event common.MapStr) {
 	if signal != nil {
 		b.pending = append(b.pending, signal)
 	}
+
+	if len(b.events) == cap(b.events) {
+		b.publish()
```

Comment:
Are bulk events only sent if the queue if completely full or also after a timeout?